services/121-service/src/registration/services/registrations-pagination.service.ts

```diff
@@ -111,21 +116,27 @@ export class RegistrationsPaginationService {
       },
     );
 
-    const registrationDataRelations =
+    const programRegistrationAttributeRelations =
       await this.programService.getAllRelationProgram(programId);
-    const registrationDataNamesProgram = registrationDataRelations
-      .map((r) => r.name)
-      .filter((r) => r !== DefaultRegistrationDataAttributeNames.phoneNumber); // Phonenumber is already in the registration table so we do not need to filter on it twice
+    const relationsWithoutPhoneNumber =
+      programRegistrationAttributeRelations.filter(
+        (r) => r.name !== DefaultRegistrationDataAttributeNames.phoneNumber,
+      );
+    const relationNamesWithoutPhonenumber = relationsWithoutPhoneNumber.map(
+      (r) => r.name,
+    );
+    // Phonenumber is already in the registration table so we do not need to filter on it twice
```

Comment:
I think some lines have been auto-formatted to the wrong position...

```suggestion
    const relationsWithoutPhoneNumber =
      programRegistrationAttributeRelations.filter(
        // <comment-should-go-here?>
        (r) => r.name !== DefaultRegistrationDataAttributeNames.phoneNumber,
      );
    // <or comment-should-go-here?>
    const relationNamesWithoutPhonenumber = relationsWithoutPhoneNumber.map(
      (r) => r.name,
    );
```

Reply:
In _this specific_ comment, I was talking about the position of the `// comment` only;

Not the variable names..
But, yes, I agree/understand your point about those. ;)